jmeter-service/main.go

```diff
@@ -99,6 +100,17 @@ func runTests(event cloudevents.Event, shkeptncontext string, data keptnv2.TestT
 	var res bool
 	healthCheckWorkload, err = getWorkload(jmeterconf, TestStrategy_HealthCheck)
 	if healthCheckWorkload != nil {
+
+		res , err = net.LookupIP(serviceUrl.String())
+		if err != nil {
+			msg := fmt.Sprintf("Cant reach URL %s from JMeter-Service: %s", serviceUrl, err.Error())
+			logger.Error(msg)
+			if err := sendErroredTestsFinishedEvent(shkeptncontext, event, startedAt, msg, logger); err != nil {
```

Comment:
Please change the function from `sendErroredTestsFinishedEvent` to `sendTestsFinishedEvent` because technically the service did not errored. It just could not reach the URL.